README.md

```diff
@@ -15,6 +15,9 @@ This is a project template for [vue-cli](https://github.com/vuejs/vue-cli). **It
 
 ``` bash
 $ npm install -g vue-cli
+# the master branch now is working with vue 2
+$ vue init webpack my-project
+# for using vue ^1.0.0 && < 2.0.0
 $ vue init webpack#1.0 my-project
```

Comment:
Lines 20 and 21 can also be removed, this is already explained in Line 5 of README